.travis.yml

```diff
@@ -10,18 +10,20 @@ go:
   - 1.8
   - 1.9
 
+go_import_path: github.com/tensorflow/k8s
+
 install:
   # get coveralls.io support
   - go get github.com/mattn/goveralls
   - go get -u github.com/alecthomas/gometalinter
   - gometalinter --install
 
 script:
+  - go build -o tf-operator github.com/tensorflow/k8s/cmd/tf_operator
```

Comment:
@jlewi When will this repo be moved to kubeflow org? This PR maybe changed after the moving.

Reply:
There will be lots of places to be changed, and we could replace the string with `github.com/kubeflow/tf-operator` globally, so the transformation does not block the PR. 😄

Reply:
We can move it as soon as the PRs adding TFJob tests to kubeflow/kubeflow are submitted.
Specifically
https://github.com/tensorflow/k8s/pull/374
https://github.com/kubeflow/kubeflow/pull/227

@gaocegege Can you take a look at those?